metaflow/metaflow_environment.py

```diff
@@ -49,8 +51,26 @@ def bootstrap_commands(self, step_name, datastore_type):
 
     def add_to_package(self):
         """
-        A list of tuples (file, arcname) to add to the job package.
-        `arcname` is an alternative name for the file in the job package.
+        Called to add custom files needed for this environment. This hook will be
+        called in the `MetaflowPackage` class where metaflow compiles the code package
+        tarball. This hook can return one of two things:
+          - a generator yielding a tuple of `(file_path, arcname)` to add files to
```

Comment:
this is a bit funky, we shouldn't return two types of tuples

Reply:
already addressed above with a question.